main.go

```diff
@@ -146,7 +147,7 @@ func startRosettaRPC(ctx context.Context, api api.FullNode) error {
 	router := newBlockchainRouter(network, asserter, api, retriever, r)
 	loggedRouter := server.LoggerMiddleware(router)
 	corsRouter := server.CorsMiddleware(loggedRouter)
-	server := &http.Server{Addr: fmt.Sprintf(":%d", ServerPort), Handler: corsRouter}
+	server := &http.Server{Addr: fmt.Sprintf(":%d", ServerPort), Handler: corsRouter} //nolint
```

Comment:
The linter is flagging an error because the http.Server configuration doesn't have a ReadHeaderTimeout set. Should we define this parameter? If so, what would be an appropriate timeout value?